### PR TITLE
📖 Story 360 docs

### LINKED ISSUE #29511: [Story 360] Public documentation
Write public documentation for `amp-story-360`.

cc @ampproject/wg-stories

### PR DESCRIPTION
**Pull request by processprocess**
Documentation for amp-story-360 for amp.dev

Closes #29511

### CODE REVIEW

**Comment by processprocess** [comment]
cc @t0mg

File: extensions/amp-story-360/0.1/amp-story-360.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+### Configuration
+
+The `amp-story-360` component accepts 1 descendant. This descendant must be an [amp-img](https://amp.dev/documentation/components/amp-img/?format=stories) or [amp-video](https://amp.dev/documentation/components/amp-video/?format=stories).
+
+#### amp-image
+
+[sourcecode:html]
+<amp-story-360 layout="fill">
+<amp-image layout="fill" src="assets/image360.jpg"></amp-image>
```
Comment (gmajoulet) [inline extensions/amp-story-360/0.1/amp-story-360.md:96]:
```suggestion
<amp-img layout="fill" src="assets/image360.jpg"></amp-img>
```

Can you run these snippets through validator.ampproject.org and make sure we didn't forget anything?

File: extensions/amp-story-360/0.1/amp-story-360.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+</amp-story-360>
+[/sourcecode]
+
+#### amp-video
+
+[sourcecode:html]
+<amp-story-360 layout="fill">
+<amp-video layout="fill" poster="assets/poster.jpg" autoplay loop>
+
+<source src="assets/video360.mp4" type="video/mp4" />
```
Comment (gmajoulet) [inline extensions/amp-story-360/0.1/amp-story-360.md:106]: Nit: indentation and line breaks

**Review by gmajoulet — APPROVED** [review]

File: extensions/amp-story-360/0.1/amp-story-360.md (hunk 1 above)
Comment (processprocess) [reply extensions/amp-story-360/0.1/amp-story-360.md]: PR #30832 and #31028 aren't reflected on validator.ampproject.org yet. Once they are these should pass validation.

Comment (processprocess) [reply extensions/amp-story-360/0.1/amp-story-360.md]: PR #30832 and #31028 aren't reflected on validator.ampproject.org yet. 
Once they are these should pass validation.

File: extensions/amp-story-360/0.1/amp-story-360.md (hunk 2 above)
Comment (processprocess) [reply extensions/amp-story-360/0.1/amp-story-360.md:106]: For some reason `gulp prettify` requires this formatting.

File: extensions/amp-story-360/0.1/amp-story-360.md (hunk 1 above)
Comment (processprocess) [reply extensions/amp-story-360/0.1/amp-story-360.md]: Those PRs are now on validator.ampproject.org and these are passing :)

File: extensions/amp-story-360/0.1/amp-story-360.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+   duration="3s">
+...
+</amp-story-360>
+[/sourcecode]
+
+The `duration` attribute is required for animation.
+
+#### Gyroscope
+
+Use the device gyroscope sensor to set the image's orientation. The component defaults to the `heading-end` value.
```
Comment (t0mg) [inline extensions/amp-story-360/0.1/amp-story-360.md:139]: nit: this sort of implies that `heading-end` is required to apply a default heading, which is not the case: if `heading-end` is not provided, `heading-start` is used. If neither are provided the heading will default to 0.

File: extensions/amp-story-360/0.1/amp-story-360.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+##### Activate 360 button
+
+If the device requires user permission to access the device’s gyroscope sensor an “activate 360” button will display.
+
+<amp-img src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/activate-360.png" layout="intrinsic" width="88.5" height="36">
+
+##### Animation fallback
+
+If the device does not support gyroscope or the user declines permission, the component will fallback to animating.
```
Comment (t0mg) [inline extensions/amp-story-360/0.1/amp-story-360.md:162]: nit: only if the animation is defined, ie if at least `duration` is provided

File: extensions/amp-story-360/0.1/amp-story-360.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+## Attributes
+
+### Animation
+
+#### heading-start {float} optional
+
+Specifies the heading (horizontal angle in degrees) at the beginning of an animation.
```
Comment (t0mg) [inline extensions/amp-story-360/0.1/amp-story-360.md:170]: nit: centered on 0 (default), with a negative angle looking towards the left, positive angle to the right

File: extensions/amp-story-360/0.1/amp-story-360.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+#### pitch-start {float} optional
+
+Specifies the pitch (vertical angle in degrees) at the beginning of an animation.
```
Comment (t0mg) [inline extensions/amp-story-360/0.1/amp-story-360.md:174]: nit: centered on 0 (default) at the horizon, with a negative angle looking downwards, positive angle upwards

**Pull request merged**